cmake/buildTargets.cmake

```diff
@@ -9,10 +9,12 @@ if (VINECOPULIB_SHARED_LIB)
 else()
     add_library(vinecopulib INTERFACE)
     target_include_directories(vinecopulib INTERFACE
-            $<BUILD_INTERFACE:${vinecopulib_includes}>
-            $<INSTALL_INTERFACE:include/vinecopulib>)
```

Comment:
Why is this removal needed?

Reply:
its not removed: I kept the BUILD_INTERFACE part here,
 then moved the INSTALL_INTERFACE outside of the if because its also needed for the library
note that I had to remove the /vinecopulib subdirectory because vinecopulib.hpp is at the root dir